src/test/integration/user/userResetPassword.ts

```diff
@@ -58,10 +58,12 @@ export default (): void =>
     })
 
     it('Verify changed password', async () => {
-      const userAuthProvider = (await UserProviderController.read({
+      const userAuthProviders = (await UserProviderController.read({
```

Comment:
just a suggestion. 
I have seen many calls to UserProviderController.read with casting. 

you can simply declare the controller and repository methods as 
```typescript
export const read = async <P>(
  props: {
    provider: AuthProvider
    user: User
  },
  client: BaseProtocol = DB
): Promise<UserAuthProvider<P>> => {
```

and invoke the method with desired P type. e.g.
```typescript
const userAuthProviders = await UserProviderController.read<AuthProviderLocalProps>({user,provider: AuthProvider.local})
```

to avoid constant casting and simplify the code.

what do you think?

Reply:
done